tests/integration_tests/full_install.sh

```diff
@@ -20,23 +20,14 @@ fi
 . $GEOIPS_PACKAGES_DIR/geoips/tests/integration_tests/base_install.sh
 . $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh ancillary_data cartopy_shapefiles $test_exit $install_script
 . $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh settings_repo .vscode $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_noaa_aws $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_multi_scan_times $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_gpm $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_sar $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_scat $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_smap $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_seviri $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_viirs $test_exit $install_script
+geoips config install test_data_noaa_aws test_data_multi_scan_times test_data_gpm test_data_sar test_data_scat test_data_smap test_data_seviri test_data_viirs test_data_amsr2 test_data_saphir test_data_arctic_weather_satellite
 
 if [[ "$include_reference_repos" == "true" ]]; then
   . $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh settings_repo .github $test_exit $install_script
   . $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh settings_repo geoips_ci $test_exit $install_script
-  # These may not currently be on cira nexcloud
+  # These may not currently be on cira nextcloud
```

Comment:
Wait,  is it nexcloud or nextcloud?

Reply:
It's nextcloud :)
![Uploading Screenshot 2025-09-08 at 9.58.09 AM.png…]()